src/domains/idpay-app/terraform.sh

```diff
@@ -1 +1 @@
-../../../scripts/terraform.sh
```

Comment:
Please from the directory of idpay-app use the command
`unlink terraform.sh`
To unlink terraform sh and after 
`ln -s ../../../scripts/terraform.sh terraform.sh`
To re create the symbolic link and push the changes.
After that we'll not see anymore this change and the idpay-app pipeline should go😉

Reply:
@lucaconsalvi  @GiovanniMancini 
I rollbacked it. could you verify if it's working?

Reply:
Great! 💪 Thank you!